src/xcode/ENA/ENAUITests/Onboarding/ENAUITestsOnboarding.swift

```diff
@@ -33,6 +33,8 @@ class ENAUITests_00_Onboarding: CWATestCase {
 
 		app.buttons["AppStrings.Onboarding.onboardingInfo_enableLoggingOfContactsPage_button"].waitAndTap()
 
+		app.buttons["AppStrings.Onboarding.onboardingContinue"].waitAndTap()
```

Comment:
Didn't get why we need to tap same button twice?

Reply:
Not the same button but on the next onboarding screen the next continue button. `test_0001_OnboardingFlow_EnablePermissions_normal_XS` does this as well.